### DISCUSSION TITLE
Welcome to dynamicprompts Discussions!

### DISCUSSION THREAD

**Discussion by adieyal** [adieyal/dynamicprompts#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thedailyhedge** [comment]
Hi, just wanted to say thanks for this great module.  I'm working on a project that's driven by random prompting, and dynamicprompts is basically critical.  I started with the basic text files, now I'm trying the YAML support.  Thanks again!
